evap/evaluation/templates/notebook.html

```diff
@@ -0,0 +1,30 @@
+<div class="notebook">
+   <div class="collapse position-fixed collapse-horizontal notebook-animation mt-0 d-print-none" tabindex="-1" id="notebook" aria-expanded="false" aria-labelledby="notebook">
+        <div class="card">
+            <div class="card-body notebook-card notebook-div">
+                <div class="offcanvas-header notebook-button mb-2">
+                    <h5 class="card-title m-0">
+                        {% trans 'Notebook' %}
+                    </h5>
+                    <button type="button" class="btn-close text-reset" data-bs-toggle="collapse" data-bs-target="#notebook" aria-controls="collapseNotebook" id="notebookButtonClose"></button>
+                </div>
+            
+                <form method="POST" action="{% url 'evaluation:notebook_edit' %}" class="notebook-div" id="notebook-form" data-state="ready">
+                    {% csrf_token %}
+                    {{ notebook_form.notes }}
+                    <div class="d-flex justify-content-center p-3">
+                        <button class="btn btn-primary notebook-button" id="notebook-save-button" type="submit" data-errormessage="{% trans 'The server is not responding.' %}">
```

Comment:
data-errormessage and `visible-if-error` are duplicated (->data-errormessage should be removed?)

Reply:
no, this is the message for the alert. discussed that with @niklasmohrin

Reply:
Still a fair point though, maybe we don't need the alert if the error message is displayed in the button already

Reply:
I think a alert is more noticable.

Reply:
in that case the point still stands and we probably don't want both. @janno42 ?

Reply:
We'll keep the alert but the button's text should be changed to the original text.